src/ray/common/ray_config_def.h

```diff
@@ -108,6 +108,18 @@ RAY_CONFIG(bool, scheduler_loadbalance_spillback,
            getenv("RAY_SCHEDULER_LOADBALANCE_SPILLBACK") != nullptr &&
                getenv("RAY_SCHEDULER_LOADBALANCE_SPILLBACK") != std::string("1"))
 
+/// Whether to use the hybrid scheduling policy, or one of the legacy spillback
+/// strategies. In the hybrid scheduling strategy, leases are packed until a threshold,
+/// then spread via weighted (by critical resource usage).
+RAY_CONFIG(bool, scheduler_hybrid_scheduling,
```

Comment:
What's happening if both this and `scheduler_loadbalance_spillback` are true?

Reply:
Looks like `scheduler_loadbalance_spillback ` is ignored. Can you add a TODO comment to clean this up?

Reply:
This flag takes precedence over the other one.